docs/source/_templates/layout.html

```diff
@@ -1,16 +1,53 @@
 {% extends "!layout.html" %}
 
+
+
 <link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html" />
+
 {% block menu %}
 <div>
   <a style="color:#F05732" href="{{ theme_canonical_url }}{{ pagename }}.html">
     You are viewing unstable developer preview docs.
     Click here to view docs for latest stable release.
   </a>
 </div>
+
 {{ super() }}
 {% endblock %}
 
+{% block sidebartitle %}
+
+{% if logo and theme_logo_only %}
+  <a href="http://pytorch.org">
```

Comment:
https please

Instead of hardcoding, please add a variable to `docs/source/conf.py` and reference it here.